includes/class-fees-api.php

```diff
@@ -68,6 +68,29 @@ public static function has_paid_plans() {
 		return $total > 0;
 	}
 
+
+    /**
+	 * Check if there are enabled plans.
+	 * This does a count for all enabled plans regardless of the amount.
+	 *
+	 * @since x.x
+	 *
+	 * @return bool
+	 */
+	public static function has_enabled_plans() {
```

Comment:
Can we reduce duplication here a bit by adding has_enabled_plans( $paid = null )

Then:
$cache_key = 'enabled_plan_count';
$query = "SELECT count(*) FROM {$wpdb->prefix}wpbdp_plans WHERE enabled != 0";
if ( $paid ) {
$cache_key = 'paid_plan_count';
$query .= "AND amount > 0";
}